packages/letspeppol-sdk-php/src/Resources/AppClient.php

```
@@ -0,0 +1,495 @@
+     * $doc = $client->app()->getDocument('doc123');
+     * echo "Invoice: {$doc['invoiceNumber']}\n";
+     * echo "From: {$doc['partnerName']}\n";
+     * echo "Total: {$doc['total']} {$doc['currency']}\n";
+     * file_put_contents('invoice.xml', $doc['ubl']);
+     * ```
+     *
+     * @param string $id Document ID
+     * @return array Complete document details with UBL content
+     * @throws ApiException When document not found (404)
+     */
+    public function getDocument(string $id): array
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)